proto/api.proto

```diff
@@ -52,6 +52,41 @@ service Api {
   };
 
   // Drift V2
+
+  rpc GetDriftPerpPositions(GetDriftPerpPositionsRequest)  returns (GetDriftPerpPositionsResponse) {
```

Comment:
one of the things that's missing here is that adding the context slot information to each endpoint:

```
All endpoints should return data in the format { context: { slot: 1234 }, data: payload } 
```

Reply:
lets create a ticket for this so i add it for all new endpoints, i can't get to it today.

Reply:
I think we might as wait on merging all of this in then. There's no rush and you'll almost certainly have little to no merge conflicts.